options_linux.cmake

```diff
@@ -62,12 +62,14 @@ if (DESKTOP_APP_SPECIAL_TARGET)
     target_link_options(common_options INTERFACE $<IF:$<CONFIG:Debug>,,-g -flto -fuse-linker-plugin>)
 endif()
 
-target_link_libraries(common_options
-INTERFACE
-    $<TARGET_OBJECTS:desktop-app::linux_jemalloc_helper>
-    $<LINK_ONLY:desktop-app::external_jemalloc>
-    ${CMAKE_DL_LIBS}
-)
+if (LINUX AND NOT DESKTOP_APP_DISABLE_JEMALLOC)
+	target_link_libraries(common_options
+	INTERFACE
+	    $<TARGET_OBJECTS:desktop-app::linux_jemalloc_helper>
+	    $<LINK_ONLY:desktop-app::external_jemalloc>
+	    ${CMAKE_DL_LIBS}
```

Comment:
You can't just remove libdl from linking of all desktop-app projects :see_no_evil: